ateam_ssl_simulation_radio_bridge/src/message_conversions.cpp

```diff
@@ -49,7 +54,7 @@ RobotControl fromMsg(const ateam_msgs::msg::RobotMotionCommand & ros_msg, int ro
     case ateam_msgs::msg::RobotMotionCommand::KICK_NOW:
     case ateam_msgs::msg::RobotMotionCommand::KICK_ON_TOUCH:
     case ateam_msgs::msg::RobotMotionCommand::KICK_ON_CAPTURE:
-      proto_robot_command->set_kick_speed(ros_msg.kick_speed);
+      proto_robot_command->set_kick_speed(ros_msg.kick_speed * 3.0);
```

Comment:
why do we do this multiplication?

Reply:
Trying to account for discrepancies between real robot kick speeds and sim robot kick speeds. 
All of this kick speed stuff is something we should double check, especially if/when we make new kicker hardware.

Reply:
seems okay for now then